### PR TITLE
Destination Postgres: `airbyte_meta`, CDK upgrade and other fixes

### PR DESCRIPTION
**Pull request by gisripa**

### CODE REVIEW

File: airbyte-cdk/java/airbyte-cdk/db-destinations/src/main/java/io/airbyte/cdk/integrations/destination/jdbc/DataAdapter.java [hunk 1]
```diff
@@ -9,6 +9,7 @@
 import java.util.function.Function;
 import java.util.function.Predicate;
 
+// TODO: Delete this class & test
```
Comment (edgao) [inline airbyte-cdk/java/airbyte-cdk/db-destinations/src/main/java/io/airbyte/cdk/integrations/destination/jdbc/DataAdapter.java:12]: what's preventing deleting this class right now?

**Review by edgao — APPROVED** [review]

File: airbyte-cdk/java/airbyte-cdk/db-destinations/src/main/java/io/airbyte/cdk/integrations/destination/jdbc/JdbcSqlOperations.java [hunk 2]
```diff
@@ -138,9 +128,7 @@ protected void writeBatchToFile(final File tmpFile, final List<PartialAirbyteMes
         final CSVPrinter csvPrinter = new CSVPrinter(writer, CSVFormat.DEFAULT)) {
       for (final PartialAirbyteMessage record : records) {
         final var uuid = UUID.randomUUID().toString();
-        // TODO we only need to do this is formatData is overridden. If not, we can just do jsonData =
```
[the remark this answers is not in the document]
Comment (gisripa) [reply airbyte-cdk/java/airbyte-cdk/db-destinations/src/main/java/io/airbyte/cdk/integrations/destination/jdbc/JdbcSqlOperations.java:141]: Yeah. implement the that with the new unified interface, similar to postgres/redshift

**Review by jbfbell — APPROVED** [review]

**Comment by gisripa** [comment]
[Pre-release sync succeeded](https://cloud.airbyte.com/workspaces/b61bc266-ef3c-460c-af2f-f70da4a2993c/connections/b0b82b70-a732-4aad-b623-13d5c5a75de2/job-history#9801555::0)

File: airbyte-integrations/connectors/destination-postgres/gradle.properties [hunk 3]
```diff
@@ -1,3 +1,5 @@
 # our testcontainer has issues with too much concurrency.
 # 4 threads seems to be the sweet spot.
 testExecutionConcurrency=4
+# large sync test takes a while, add 15m timeout.
+JunitMethodExecutionTimeout=15 m
```
Comment (stephane-airbyte) [inline airbyte-integrations/connectors/destination-postgres/gradle.properties:5]: please don't do a blanket execution timeout bump. You can use the @Timeout on a single function to do the same thing

**Pull request merged**